templates/vue/src/utils/Helpers.js

```diff
@@ -432,7 +432,7 @@ export default class Helpers {
    * 3. The tooltip is clipped on BOTH the bottom and the right (when the
    *    connection is on the bottom-right of the CoS)
    */
-  static positionTooltip(target, tooltip, container) {
+  static positionTooltip(target, tooltip, container, { dryRun = false } = {}) {
```

Comment:
What is `dryRun`? Could we include something in the comment block above about this?